settings.gradle

```diff
@@ -49,3 +51,9 @@ include 'clients',
     'streams:upgrade-system-tests-26',
     'streams:upgrade-system-tests-27',
     'tools'
+
+buildCache {
```

Comment:
I like the idea of keeping this in settings.gradle since it's more "discoverable", and thus easier to see what is and isn't enabled caching-wise.  Settings in gradle.properties tend to be forgotten about.

But to keep the setting in one place, I'll remove it from gradle.properties.

Reply:
It seemed better to have this in `gradle.properties` since we define the parallelism there too. Generally, no-one ever looks at `settings.gradle` as far as I can tell (i.e. it's not more discoverable).